app/src/main/java/com/celzero/bravedns/ui/ExcludeAppDialog.kt

```diff
@@ -23,43 +23,27 @@ import android.util.Log
 import android.view.View
 import android.view.Window
 import android.view.WindowManager
-import android.widget.Button
 import android.widget.CompoundButton
-import android.widget.ImageView
-import android.widget.TextView
-import androidx.appcompat.widget.AppCompatCheckBox
 import androidx.appcompat.widget.SearchView
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.celzero.bravedns.R
 import com.celzero.bravedns.database.AppInfoRepository
 import com.celzero.bravedns.database.CategoryInfoRepository
+import com.celzero.bravedns.databinding.ExcludeAppDialogLayoutBinding
 import com.celzero.bravedns.service.PersistentState
 import com.celzero.bravedns.ui.HomeScreenActivity.GlobalVariable.DEBUG
 import com.celzero.bravedns.util.Constants.Companion.LOG_TAG
 import com.celzero.bravedns.viewmodel.ExcludedAppViewModel
 import com.google.android.material.chip.Chip
-import com.google.android.material.chip.ChipGroup
 import java.util.stream.Collectors
 
 
-class ExcludeAppDialog(private var activity: Context,
-                       private val appInfoRepository: AppInfoRepository,
-                       private val categoryInfoRepository: CategoryInfoRepository,
-                       private val persistentState:PersistentState,
-                       internal var adapter: RecyclerView.Adapter<*>,
-                       var viewModel: ExcludedAppViewModel) : Dialog(activity),
-    View.OnClickListener, SearchView.OnQueryTextListener {
+class ExcludeAppDialog(private var activity: Context, private val appInfoRepository: AppInfoRepository, private val categoryInfoRepository: CategoryInfoRepository, private val persistentState: PersistentState, internal var adapter: RecyclerView.Adapter<*>, var viewModel: ExcludedAppViewModel) : Dialog(activity), View.OnClickListener, SearchView.OnQueryTextListener {
+    private lateinit var b: ExcludeAppDialogLayoutBinding
```

Comment:
Repeat: `b` needn't be set to `null` ever again?

Reply:
Nope. 
These are not fragments, only fragment bindings are required to be set to `null`.